altos-rust/altos-core/src/test.rs

```diff
@@ -51,12 +51,6 @@ pub fn current_task() -> Option<&'static mut TaskControl> {
     unsafe { CURRENT_TASK.as_mut().map(|task| &mut ***task) }
 }
 
-pub fn create_two_tasks() -> (TaskHandle, TaskHandle) {
```

Comment:
This was being used in several tests, I think it's a common enough pattern in the tests we're running to keep it. You also didn't remove it from any of the tests themselves so currently trying to compile for testing errors.